core/server/common/src/main/java/alluxio/cli/Format.java

```diff
@@ -82,6 +82,7 @@ public static void main(String[] args) {
       LOG.error("Failed to format", e);
       System.exit(-1);
     }
+    LOG.info("Format completes");
```

Comment:
nit: I think "Formatting complete" sounds better since the per-journal messages say "Formatting {}"